spk/jackett/Makefile

```diff
@@ -28,14 +26,24 @@ SERVICE_PORT_TITLE = Jackett(HTTP)
 # Admin link
 ADMIN_PORT = $(SERVICE_PORT)
 
-POST_STRIP_TARGET = jackett_extra_install
-
-REQUIRED_DSM = 5.0
-# no PPC support for mono, see cross/mono/Makefile for details
 UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 
-BUSYBOX_CONFIG = usrmng daemon
-ENV += BUSYBOX_CONFIG="$(BUSYBOX_CONFIG)"
+# Mono build for unsupported .NET archs
+include ../../mk/spksrc.archs.mk
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS)),$(ARCH))
+DEPENDS = cross/$(SPK_NAME)-mono
+SERVICE_SETUP = src/service-setup-mono.sh
+endif
+ifeq ($(findstring $(ARCH),$(i686_ARCHS)),$(ARCH))
+DEPENDS = cross/$(SPK_NAME)-mono
+SERVICE_SETUP = src/service-setup-mono.sh
+endif
+ifeq ($(findstring $(ARCH),$(ARMv7L_ARCHS)),$(ARCH))
+DEPENDS = cross/$(SPK_NAME)-mono
+SERVICE_SETUP = src/service-setup-mono.sh
+endif
```

Comment:
Same statements. Please use a single "ifeq" condition with all of them, and include ChangeLog here